src/status_im/multiaccounts/key_storage/core.cljs

```diff
@@ -7,6 +7,8 @@
             [utils.i18n :as i18n]
             [status-im.keycard.backup-key :as keycard.backup]
             [status-im.keycard.common :as common]
+            [status-im.ui.components.react :as react]
```

Comment:
@teoarjun can you not use this import from here and instead add linking in namespace `react-native.core` and import from there in this file instead. 

Eventually this file will be moved to status_im2 or some namespace outside of status_im and it would be better to not create more work for that migration 👍 

cc @flexsurfer

Reply:
yes but first we need to know why linking is used here

Reply:
had to add linking due to figma UI , check screenshot here - https://github.com/status-im/status-mobile/pull/16351#discussion_r1239336483
Also @J-Son89  , I tried to use `react-native.core` instead of  `status-im.ui.components.react :as react` , but it didn't work for me. Threw error

Reply:
Thanks @teoarjun, what is the error? Let's try to figure this out as using from `status-im...` will just lead to someone else needing to refactor later when we can ideally avoid that now 👌

Reply:
It's worked now, thanks